jest.config.js

```diff
@@ -49,10 +49,10 @@ module.exports = {
   ],
   coverageThreshold: {
     "global": {
-      "branches": 80,
-      "functions": 90,
-      "lines": 90,
-      "statements": 90
+      "branches": 88,
+      "functions": 97,
```

Comment:
I would not raise the threshold above 90, 95 tops -- it will make it really difficult to make changes in the future...